### DISCUSSION TITLE
compression/decompression other than tensor_codec

### DISCUSSION THREAD

**Discussion by dskhanirfan** [securefederatedai/openfederatedlearning#431] [General]
Hi 
Is there any other compression (at aggregator) decompression (at colab) at available in openFL other than tensor_codec? If I need to do a custom compression decompression method, which functions should I change? Is there any example available how to add custom compression/decompression in openFL?

**Comment by vivektalwar689** [comment]
I don't think currently we have other such compression-decompression method available other than tensor_codec
